__tests__/server.test.js

```diff
@@ -0,0 +1,46 @@
+const request = require('supertest')
+const app = require('../server');
+const db = require('../db/connection');
+const data = require('../db/data/test-data/index');
+const seed = require('../db/seeds/seed');
+
+beforeAll(() => {
+    return seed(data).then();
+})
+
+afterAll(() => {
+    return db.end();
+});
+
+describe('app', () => {
+    test('tests the server connection', () => {
+        return request(app)
+            .get('/api')
+            .expect(200)
+            .then();
+    })
+    test('tests the database connection', () => {
```

Comment:
You have two describe blocks which seem to be checking the same thing, why do the first if you're essentially re-checking it in the second. I'd get rid of the first if that's the case.

Reply:
I've removed the first test as suggested and tweaked the description in the second test.